mlrun/db/httpdb.py

```diff
@@ -4224,6 +4227,8 @@ def store_alert_config(
             if isinstance(alert_data, AlertConfig)
             else AlertConfig.from_dict(alert_data)
         )
+        # The validation must be here because the user can call this function directly
+        # via `mlrun.get_run_db().store_alert_config()`
```

Comment:
```suggestion
    # Validation is necessary here because users can directly invoke this function
    # through `mlrun.get_run_db().store_alert_config()`.

```